tests/conftest.py

```diff
@@ -1,14 +1,13 @@
 import os
 import shutil
 import subprocess
+import venv as virtualenv
```

Comment:
Super nit, but I'd rather rename the variable than alias the import here.

Reply:
Fair! I got lazy and aliased it, but renaming is definitely better.